pytransifex/api.py

```diff
@@ -175,21 +179,23 @@ def update_source_translation(
                     logging.info(f"Source updated for resource: {resource_slug}")
                     return
 
-        raise Exception(
+        raise ValueError(
             f"Unable to find resource '{resource_slug}' in project '{project_slug}'"
         )
 
     @ensure_login
     def get_translation(
         self,
+        *,
         project_slug: str,
         resource_slug: str,
         language_code: str,
-        output_dir: Path,
-    ):
+        path_to_output_dir: str,
```

Comment:
**Potential compatibility issue**: `output_dir` renamed to `path_to_output_dir` for the sake of expliciteness.